edenai_apis/apis/corticalio/corticalio_api.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict
+
+from edenai_apis.apis.corticalio.client import CorticalClient
+from edenai_apis.apis.corticalio.helpers import normalize_keywords
+from edenai_apis.features import TextInterface
+from edenai_apis.features.provider.provider_interface import ProviderInterface
+from edenai_apis.features.text import KeywordExtractionDataClass
+from edenai_apis.loaders.data_loader import ProviderDataEnum
+from edenai_apis.loaders.loaders import load_provider
+from edenai_apis.utils.types import ResponseType
+
+
+class CorticalioApi(
+    ProviderInterface,
+    TextInterface
+):
+    provider_name = "corticalio"
```

Comment:
upper case constant? or was there a reason why we chose lower case
```suggestion
    PROVIDER_NAME = "corticalio"
```

Reply:
Yes, it has to be lower case or some tests from Eden are going to fail